Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -5,13 +5,25 @@ extension Style {
 
     /// This function creates an expression that will localize the `textField` property of a `SymbolLayer`
     /// - Parameter locale: A `SupportedLanguage` based `Locale`
-    internal func localizeLabels(into locale: Locale) throws {
+    public func localizeLabels(into locale: Locale, for layerIds: [String]? = nil) throws {
+
+        /// Do nothing if we do not support the locale
+        if !supportedLocales.contains(locale) {
```

Comment:
This appears to be the only place where `supportedLocales` is referenced. Rather than declaring it as a global constant, declare it in this method as a local constant to avoid polluting the global namespace.

Reply:
Also, what do you think of making `supportedLocales` an array of locale identifiers instead of an array of locales? Then this line would become
```suggestion
        if !supportedLocaleIdentifiers.contains(locale.identifier) {
```

Reply:
also, we should think carefully about whether we mean locale identifier or [language code](https://developer.apple.com/documentation/foundation/locale/2293346-languagecode).

Reply:
Is there any value in exposing the supported locale identifiers publicly? It seems like it could be beneficial to provide clients with strongly-typed values to use rather than having them manually specify locale ids (strings) and handle exception (or undefined behavior) if/when they aren't supported. I just realized that I am essentially making an argument in favor of the enum that was removed below. I have no insight into what prompted these changes, or some of the specific use cases, so take this with a grain of salt

Reply:
Also, couldn't folks be using custom tile sources that support locales outside of our pre-defined set? (How) Do we handle that case?

Reply:
I'm not sure what the requirements are around that. But that would be another ticket entirely given that is different than Android and hydrogen